app/views/unattended/provisioning_templates/snippet/kickstart_rhsm.erb

```diff
@@ -4,7 +4,25 @@ name: kickstart_rhsm
 model: ProvisioningTemplate
 snippet: true
 description: |
-  Generate Anaconda RHSM configuration for Red Hat OS registration
+  Generate Anaconda RHSM configuration for Red Hat OS registration available on Red Hat 9 or newer
+
+  Parameters for use with subscription-manager (Red Hat CDN, Satellite, or Katello):
+    activation_key = <key>                      Activation key string
+    subscription_manager = 'true'               You're going to use subscription-manager
+    subscription_manager_org = <org name>       Organization name, if required
+    http-proxy = <host>                         Proxy hostname to be used for registration
```

Comment:
Note to myself: `http-*` params don't have underscore, same as in [redhat_register.erb](https://github.com/theforeman/foreman/blob/develop/app/views/unattended/provisioning_templates/snippet/redhat_register.erb)